labgrid/driver/power/eaton.py

```diff
@@ -0,0 +1,66 @@
+from pysnmp import hlapi
```

Comment:
This pysnmp dependency needs to be added to one of the requirements files.
However, I'd much rather have this switched over to the snmpwalk binary, instead of using the python module.

Reply:
@Emantor You probably mean `snmpget`/`snmpset` as used in https://github.com/labgrid-project/labgrid/blob/master/labgrid/driver/power/sentry.py.
If the driver works well as is and because we already have https://github.com/labgrid-project/labgrid/blob/master/snmp-requirements.txt, I wouldn't insist on switching to calling the external binaries.
@b2vn Did you test it with the version in the `snmp-requirements.txt`?

Reply:
Ah, nevermind. I was under the impression that we did not have a pysnmp dependency already, but this was added with EthernetPort support (I always forget about that). In this case this is fine and should work.